patches/tModLoader/Terraria/ID/TileID.cs.patch

```diff
@@ -32,6 +32,14 @@
  			public static bool[] Corrupt = Factory.CreateBoolSet(23, 25, 112, 163, 398, 400);
  			public static bool[] Hallow = Factory.CreateBoolSet(109, 117, 116, 164, 402, 403, 115);
  			public static bool[] Crimson = Factory.CreateBoolSet(199, 203, 234, 200, 399, 401, 205);
+@@ -195,6 +_,7 @@
+ 			public static bool[] IgnoredInHouseScore = Factory.CreateBoolSet(4, 3, 73, 82, 83, 84);
+ 			public static bool[] SpreadOverground = Factory.CreateBoolSet(2, 23, 32, 60, 70, 109, 199, 352, 477, 492);
+ 			public static bool[] SpreadUnderground = Factory.CreateBoolSet(23, 109, 199, 60, 70);
++			public static bool[] CanDropFromRightClick = Factory.CreateBoolSet(4);
```

Comment:
Can there be partial nested classes? If so, this should be in one. FYI, the convention for partials is now X.TML.cs